clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsMembershipManager.java

```diff
@@ -833,12 +838,20 @@ private CompletableFuture<Void> revokeActiveTasks(final SortedSet<StreamsAssignm
 
     private CompletableFuture<Void> assignTasks(final SortedSet<StreamsAssignmentInterface.TaskId> activeTasksToAssign,
                                                 final SortedSet<StreamsAssignmentInterface.TaskId> ownedActiveTasks,
-                                                final SortedSet<StreamsAssignmentInterface.TaskId> standbyTasksToAssign) {
-        log.info("Assigning active tasks {} and standby tasks {}",
+                                                final SortedSet<StreamsAssignmentInterface.TaskId> standbyTasksToAssign,
+                                                final SortedSet<StreamsAssignmentInterface.TaskId> warmupTasksToAssign) {
+        log.info("Assigning " +
```

Comment:
nit: Should this be `info` or `debug` ? I don't remember from previous PRs so maybe info here is good.

Reply:
I think (I am not sure I remember correctly), I added this at `info` level because there is a similar message in the `revokeActiveTasks()`.  I will keep it at `info` for now and we can change it later.